src/core/src/Window/WindowResolution.cpp

```diff
@@ -26,7 +26,7 @@ const sf::VideoMode &WindowResolution::GetCurrentResolution()
 
 const std::string WindowResolution::GetCurrentResolutionString()
 {
-  boost::format fmt = boost::format("%1% x %2%") 
+  std::format fmt = boost::format("%1% x %2%") 
```

Comment:
czemu boost::format kiedy usunąłeś include boost/...? To się wgl kompiluje? Chyba lepiej przypisać do fmt std::format?